sql/core/src/test/resources/sql-tests/results/subquery/negative-cases/subq-input-typecheck.sql.out

```diff
@@ -132,15 +132,6 @@ WHERE
                            t5c
                     FROM t5)
 -- !query 9 schema
-struct<>
+struct<t4a:double,t4b:string,t4c:string>
 -- !query 9 output
-org.apache.spark.sql.AnalysisException
-cannot resolve '(named_struct('t4a', t4.`t4a`, 't4b', t4.`t4b`, 't4c', t4.`t4c`) IN (listquery()))' due to data type mismatch: 
-The data type of one or more elements in the left hand side of an IN subquery
-is not compatible with the data type of the output of the subquery
```

Comment:
Can we change the query a little bit to make it still fail? This is `negative-cases/subq-input-typecheck.sql`.

Reply:
thanks, I have modified them.